Sources/WebUI/WebViewProxy.swift

```diff
@@ -88,6 +88,13 @@ public final class WebViewProxy: ObservableObject {
         webView?.load(URLRequest(url: url))
```

Comment:
`WKWebView` only provides `load(URLRequest)` and does not offer `load(URL)`.
Do you think it would be better to align the interface of WebUI accordingly?

Reply:
I thought that might be true.
In that case, the WebView init argument should also be a URLRequest, not a URL, right?

Reply:
That feels more natural to me.

Reply:
Replaced `URL` with `URLRequest` in [be0dd24](https://github.com/cybozu/WebUI/pull/5/commits/be0dd24a30ca8f9b26b4e95ca4d465eb15e32394)